config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigquerydatasets.bigquery.cnrm.cloud.google.com.yaml

```diff
@@ -226,86 +242,71 @@ spec:
+                  removed before the table expires, or if you provide an explicit
+                  expirationTime when creating a table, that value takes precedence
+                  over the default expiration time indicated by this property.
+                format: int64
                 type: integer
               description:
-                description: A user-friendly description of the dataset.
+                description: Optional. A user-friendly description of the dataset.
                 type: string
               friendlyName:
-                description: A descriptive name for the dataset.
+                description: Optional. A descriptive name for the dataset.
                 type: string
               isCaseInsensitive:
-                description: |-
-                  TRUE if the dataset and its table names are case-insensitive, otherwise FALSE.
-                  By default, this is FALSE, which means the dataset and its table names are
-                  case-sensitive. This field does not affect routine references.
+                description: Optional. TRUE if the dataset and its table names are
+                  case-insensitive, otherwise FALSE. By default, this is FALSE, which
+                  means the dataset and its table names are case-sensitive. This field
+                  does not affect routine references.
                 type: boolean
               location:
-                description: |-
-                  Immutable. The geographic location where the dataset should reside.
-                  See [official docs](https://cloud.google.com/bigquery/docs/dataset-locations).
-
-
-                  There are two types of locations, regional or multi-regional. A regional
-                  location is a specific geographic place, such as Tokyo, and a multi-regional
-                  location is a large geographic area, such as the United States, that
-                  contains at least two geographic places.
-
-
-                  The default value is multi-regional location 'US'.
-                  Changing this forces a new resource to be created.
+                description: The geographic location where the dataset should reside.
+                  See https://cloud.google.com/bigquery/docs/locations for supported
+                  locations.
                 type: string
               maxTimeTravelHours:
-                description: Defines the time travel window in hours. The value can
-                  be from 48 to 168 hours (2 to 7 days).
+                description: Optional. Defines the time travel window in hours. The
+                  value can be from 48 to 168 hours (2 to 7 days). The default value
+                  is 168 hours if this is not set.
                 type: string
               projectRef:
-                description: The project that this resource belongs to.
+                description: The project that this resource belongs to. optional.
```

Comment:
This seems to be wrong. The projectRef is required.

Reply:
From our documentation, the projectRef under Spec seems to be optional https://cloud.google.com/config-connector/docs/reference/resource-docs/bigquery/bigquerydataset#spec

Reply:
Do you mind verifying that what happens if the `spec.projectRef` is not provided against the real GCP?

Reply:
We currently have two testcases that created bigquerydataset without a projectRef, I have run it against real GCP and working.
Also verified with a real CC cluster, created a bigquerydataset without projectRef, with only name and namespace through `kubectl apply`, and it's status is upToDate Ready.
Bigquerydataset yaml file.
```
apiVersion: bigquery.cnrm.cloud.google.com/v1beta1
kind: BigQueryDataset
metadata:
  name: bigquerydatasetsampletest
  namespace: config-control
spec:
  friendlyName: bigquerydataset-sample
```
Resource status:
```
$ kubectl get bigquerydataset -n config-control
NAME                        AGE   READY   STATUS     STATUS AGE
bigquerydatasetsampletest   97s   True    UpToDate   96s
```